crates/server/src/store/sqlite/mod.rs

```diff
@@ -948,17 +971,7 @@ impl Authority for SqliteAuthority {
         self.verify_prerequisites(update.prerequisites()).await?;
         self.pre_scan(update.updates()).await?;
 
-        let updated = self.update_records(update.updates(), true).await;
-
-        #[cfg(feature = "metrics")]
```

Comment:
Did this move elsewhere? Please isolate the corresponding removals/additions in a separate commit.

Reply:
The `zone_records_dynamically_updated` metrics was previously a single metric. This was refined to have now:
- `zone_records_dynamically_added` -> `self.metrics.persistent.dynamically_add()`
- `zone_records_dynamically_deleted` -> `self.metrics.persistent.dynamically_delete()`
- `zone_records_dynamically_updated` -> `self.metrics.persistent.dynamically_update()`

It didn't directly move elsewhere, it was removed and refined details added.

Reply:
I've split the commit into three, so changes clearer in the history.